src/questions/decentralized_exchanges.yaml

```diff
@@ -0,0 +1,786 @@
+    - A
+    - C
+    - D
+  subTopics:
+    - crypto-exchange
+  difficultyLevel: Medium
+  choices:
+    - content: Faster trade matching and settlement
+      key: A
+    - content: Slow trade matching and settlement
+      key: B
+    - content: No fees for cancelled orders
+      key: C
+    - content: No censorship resistance
+      key: D
+- uuid: 4985ad13-5ce6-4d04-8d66-96b378c13438
+  type: MultipleChoice
+  content: CEX with Onchain Order Book Model has
+  hint: Slower, Censored
+  explanation: Due to the Decentralized and Distributed Ledger storage system they are slower as the validator/miner action is required in trade execution and have to pay gas fees for transactions hence, fees are required for canceled orders, also all types of traders’ data are censorship resistance.
+  answerKeys:
+    - A
+    - D
+  subTopics:
+    - crypto-exchange
+  difficultyLevel: Medium
+  choices:
+    - content: Slow trade matching and settlement
+      key: A
+    - content: No fees for cancelled orders
+      key: B
+    - content: No Censorship resistance
+      key: C
+    - content: Censorship resistance
+      key: D
+- uuid: 39cd2b05-81f4-4b0a-a540-e1d21fa275b7
+  type: SingleChoice
+  content: ____ are basic sets of rules that allow data and cryptocurrencies to be shared securely between different nodes (i.e computers) in the blockchain network.
+  hint: NoHint
+  explanation: A protocol is a set of rules and instructions that needs to be followed by each entity of a blockchain system for secure transfer.
+  answerKeys:
+    - B
+  subTopics:
+    - crypto-exchange
+  difficultyLevel: Low
+  choices:
+    - content: Liquidity Pools
+      key: A
+    - content: Protocol
+      key: B
+    - content: Order Book
+      key: C
+    - content: Exchanges
+      key: D
+- uuid: 4f6e29dd-b146-4601-85b9-92c648310b15
+  type: SingleChoice
+  content: DEX stands for
+  hint: NoHint
+  explanation: Decentralized Exchange is a marketplace for trading crypto assets governed by protocols & nodes on the blockchain, and not by any centralized authority.
+  answerKeys:
+    - D
+  subTopics:
+    - dex-definition
+  difficultyLevel: Low
+  choices:
+    - content: Deterministic Expansion
+      key: A
+    - content: Deterministic Exchange
+      key: B
+    - content: Decentralized Expansion
+      key: C
+    - content: Decentralized Exchange
+      key: D
+- uuid: 4b669837-4d58-4ca9-abfc-4dc7378116d8
+  type: SingleChoice
+  content: What is a Decentralized Exchange?
+  hint: Buyer-Seller
+  explanation: A platform/marketplace where buyer and seller trade crypto assets without the involvement of exchange as well as any third-party intermediary.
+  answerKeys:
+    - B
+  subTopics:
+    - dex-definition
+  difficultyLevel: Low
+  choices:
+    - content: A peer-exchange-peer marketplace for crypto traders without any intermediary intervention
+      key: A
+    - content: A peer-to-peer marketplace for crypto traders without any intermediary intervention
```

Comment:
Uniswap seems to be `A peer-exchange-peer marketplace for crypto traders with intermediary interventions` but is a Decentralized Exchange. `intermediary` can be a protocol